benchmarks/model_benchmarks/model_validate/benchmark.py

```diff
@@ -0,0 +1,19 @@
+from .models import Song
+from ...utils import bench_setup
+
+class ModelValidate:
+
+    def setup(self):
+        bench_setup()
+        self.model = Song(name="abc")
+    
+    def time_model_validate(self):
+        self.model.full_clean()
```

Comment:
It would be interesting to see what impact all of the `self.model` calls have. 

If the lookup is a material amount of the time taken then it could be worth doing the lookup once so get closer to just calling `full_clean()`. 

Does that make sense?

Reply:
I had initially added a single model.full_clean() call in the PR I made to the djangobench repo, but after seeing multiple calls in forms.FormsBenchmark.time_full_clean() in this repo I thought that multiple calls were required for the benchmark, I have reverted it to a single call.

Reply:
Appologies, what I meant was that the `self.model` has a lookup to the object each time. Instead we could do. 

``` python
model = self.model
model.full_clean()
model.full_clean()
...
```

Maybe it's not worth it.